tests/data/dataset_readers/srl_dataset_reader.py

```diff
@@ -0,0 +1,39 @@
+# pylint: disable=no-self-use,invalid-name
```

Comment:
Duh... This was when I refactored to the new name, so the test does actually pass.